crates/udd-sketch/src/lib.rs

```diff
@@ -290,7 +290,7 @@ impl UDDSketch {
         assert!(
             (self.gamma.powf(1.0 / f64::powi(2.0, self.compactions as i32)) - 
              other.gamma.powf(1.0 / f64::powi(2.0, other.compactions as i32))
-            ).abs() < f64::EPSILON
+            ).abs() < 1e-9
```

Comment:
I think a little explanatory comment here would go a long way, and include the bug number.

Reply:
e.g. `// f64::EPSILON too small, see issue #396`? 

I tried to make the commit message explanatory, since I usually rely on `git blame` for this stuff, but take your point that `1e-9` by itself doesn't give much context for this threshold.

Reply:
That comment is great: succinct and pointing to more detail.